src/disk.hpp

```diff
@@ -139,22 +139,22 @@ class FileDisk : public Disk {
         } while (amtwritten != length);
     }
 
-    inline std::string GetFileName() override { return filename_.string(); }
+    std::string GetFileName() override { return filename_.string(); }
 
-    inline uint64_t GetWriteMax() const noexcept { return writeMax; }
+    uint64_t GetWriteMax() const noexcept { return writeMax; }
 
-    inline void Truncate(uint64_t new_size) override
```

Comment:
all these `inline` are redundant. The function bodies are already defined inline